src/nodes/typeGuards/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from "./alias";
+export * from "./compound";
+export * from "./simple";
+export * from "./union";
```

Comment:
[Praise] I like how you organized these 🙂 makes sense and is helpful for thinking about the way the functions can be grouped!